7-module/2-task/index.js

```diff
@@ -1,6 +1,60 @@
-import createElement from '../../assets/lib/create-element.js';
-
 export default class Modal {
   constructor() {
+    this.open();
+  }
+
+  open() {
+    let modalWindow = this.#createElem();
+
+    document.body.append(modalWindow);
+    document.body.classList.add('is-modal-open');
+    modalWindow.querySelector('.modal__close').addEventListener('click', this.close);
+    document.addEventListener('keydown', this.#keyDownEventListener);
+  }
+
+  setTitle(title) {
+    document.body.querySelector('.modal__title').textContent = title;
+  }
+
+  setBody(elem) {
+    document.body.querySelector('.modal__body').innerHTML = elem.outerHTML;
```

Comment:
`this.#modal.querySelector('.modal__body').innerHTML = elem.outerHTML;`

Reply:
done.